manifests/u/Upscayl/Upscayl/2.15.0/Upscayl.Upscayl.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Upscayl.Upscayl
+PackageVersion: 2.15.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+UpgradeBehavior: install
+ReleaseDate: 2024-12-25
+AppsAndFeaturesEntries:
+- DisplayName: Upscayl 2.15.0
+  Publisher: Nayam Amarshe
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```

This is a _Nullsoft_ installer file and `winget.exe` provides full support of it.

Is it a `x64` arch file actually? But `file.exe` from _Git Bash for Windows_ inspect it as a PE32 file for Intel 80386.